config/jobs/kubernetes-csi/external-snapshotter/external-snapshotter-config.yaml

```diff
@@ -183,7 +183,7 @@ presubmits:
     optional: true
     decorate: true
     skip_report: false
-    skip_branches: ["^(k8s_1.12.0-beta.1|release-0.4|release-1.0)$"]
+    skip_branches: ["^(k8s_1.12.0-beta.1|release-0.4|release-1.0|release-2.1)$"]
```

Comment:
These files are generated automatically. The https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-csi/gen-jobs.sh script has to be updated to exclude branches on a per-job basis.

There's currently no mechanism in the script to configure that and I don't have a good idea for how to add it.

/hold

Reply:
Sorry I don't think skipping the job entirely is the right fix. Older releases using the beta api should still function. Can we fix the kubernetes e2e version to 1.19?

Reply:
That should work. This could go into .prow.sh as in https://github.com/kubernetes-csi/external-snapshotter/pull/471

Reply:
Thanks @pohly . Can you file a new PR that is based on kubernetes-csi/external-snapshotter#471?